### ISSUE TITLE
message.completion returns ''

### ISSUE THREAD

**Issue by sobrienti** [Explosion-Scratch/claude-unofficial-api#42]
Hello, 

Since this morning every time I try to use the sendMessage method, regardless of the actual message sent, message.completion returns an empty string ''. While logging onto claude.ai, I can see that Claude responded to the prompt just fine with text. 

````
const { Claude } = await import("claude-ai");
  const claude = new Claude({
    sessionKey: [my-session-key],
  });

  await claude.init();

 const conversation = await claude.startConversation(`[my initial conversion here]`)
 const message = await conversation.sendMessage(`[my message here]`)
console.log(message.completion)
````

And message.completion: 

![image](https://github.com/Explosion-Scratch/claude-unofficial-api/assets/5480968/13ada53a-773d-4623-a148-85e34073c742)

Is there an issue with the package?

**Comment by devspelunk** [comment]
I am also dealing with this issue today.

**Comment by alessandro-bonfatti-pv** [comment]
Same for me

**Comment by immatt2015** [comment]
The reason. #41

**Comment by Explosion-Scratch** [comment]
Fixed in 91426c6
